src/librustc_trans/back/write.rs

```diff
@@ -307,7 +307,7 @@ pub struct CodegenContext<'a> {
 }
 
 struct HandlerFreeVars<'a> {
-    llcx: ContextRef,
+    _llcx: ContextRef,
```

Comment:
This field is no longer used. I left it for the moment in case it is needed again in the future, however I can remove it if desired.